### DISCUSSION TITLE
Request for Clarification on GitHub Terms of Service

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132057] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi All, I have a specific question about compliance with GitHub's terms for a project I'm working on. Is there any way to discuss this privately? Thanks!

**Comment by RachidZahrani** [comment, marked as answer]
Contact GitHub Support: Go to [GitHub Support](https://support.github.com/contact) for direct help.

Check Legal Resources: Review [GitHub’s Terms of Service](https://github.com/site/terms) and [Legal Resources](https://github.com/about/legal).

Ask in GitHub Community: Post on the [GitHub Community Forum](https://github.community/) for general advice.

Consult a Lawyer: For complex issues, consider a legal professional specializing in intellectual property or software licensing.
